gallery/gallery/lib/demos/material/banner_demo.dart

```diff
@@ -0,0 +1,120 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:gallery/l10n/gallery_localizations.dart';
+
+// BEGIN bannerDemo
+
+enum BannerDemoAction {
+  reset,
+  showMultipleActions,
+  showLeading,
+}
+
+class BannerDemo extends StatefulWidget {
+  @override
+  _BannerDemoState createState() => _BannerDemoState();
+}
+
+class _BannerDemoState extends State<BannerDemo> {
+  static const _numItems = 20;
+  var _displayBanner = true;
+  var _showMultipleActions = true;
+  var _showLeading = true;
+
+  void handleDemoAction(BannerDemoAction action) {
+    setState(() {
+      switch (action) {
+        case BannerDemoAction.reset:
+          _displayBanner = true;
+          _showMultipleActions = true;
+          _showLeading = true;
+          break;
+        case BannerDemoAction.showMultipleActions:
+          _showMultipleActions = !_showMultipleActions;
+          break;
+        case BannerDemoAction.showLeading:
+          _showLeading = !_showLeading;
+          break;
+      }
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final banner = MaterialBanner(
+      content: Text(GalleryLocalizations.of(context).bannerDemoText),
+      leading: _showLeading
+          ? const CircleAvatar(child: Icon(Icons.access_alarm))
```

Comment:
Set the `backgroundColor` of the `CircleAvatar` to Theme.of(context).colorScheme.primaryColor

Reply:
Done